processor/tasks/CROWNRun.py

```diff
@@ -50,6 +50,7 @@ def create_branch_map(self):
             nick=self.nick,
             era=self.era,
             sample_type=self.sample_type,
+            production_tag=self.production_tag,
```

Comment:
I think this task definition could be somewhat streamlined by either using the .req():
```python
dataset = ConfigureDatasets.req(self)
```
or by directly using the already defined task:
```python
dataset = self.workflow_requires()["dataset"]
```

Reply:
Yes, this works, thanks! I changed it.